cmd/trust-manager/app/app.go

```diff
@@ -95,48 +94,34 @@ func NewCommand() *cobra.Command {
 				Cache: cache.Options{
 					ReaderFailOnMissingInformer: true,
 					ByObject: map[client.Object]cache.ByObject{
+						// Cache metadata for ConfigMaps cluster-wide
+						&metav1.PartialObjectMetadata{
+							TypeMeta: metav1.TypeMeta{
+								Kind:       "ConfigMap",
+								APIVersion: "v1",
+							},
+						}: {},
 						&trustapi.Bundle{}:  {},
 						&corev1.Namespace{}: {},
+						&corev1.ConfigMap{}: {
+							// Only cache full ConfigMaps in the "watched" namespace.
+							Namespaces: map[string]cache.Config{
+								opts.Bundle.Namespace: {},
+							},
+						},
 						&corev1.Secret{}: {
 							// Only cache full Secrets in the "watched" namespace.
 							Namespaces: map[string]cache.Config{
 								opts.Bundle.Namespace: {},
 							},
 						},
-						&corev1.ConfigMap{}: {
-							// Watch full config maps across all namespaces.
-							// TODO: create a seperate cache for targets and sources and only
```

Comment:
I'm surprised that this comment is still present in the codebase. Since it has been fixed &a separate cache was added.